client/packages/common/src/intl/locales/en/common.json

```diff
@@ -833,6 +833,7 @@
   "label.doses-plural_one": "dose",
   "label.doses-plural_other": "doses",
   "label.doses-received": "Doses Received",
+  "label.doses-short": "dse",
```

Comment:
abbreviation for doses is `dos.`

Reply:
If real estate is an issue could also potentially consider `d.` 

Users should be able to get it from the context I think?

Reply:
If real estate is an issue could also potentially consider `d.` 

Users should be able to get it from the context I think?